client/src/App.css

```diff
@@ -0,0 +1,28 @@
+.App {
+  text-align: center;
```

Comment:
These App files are what make up the Component. We'll delete this in the next pull request though so we can start a new one by ourselves. From scratch.